javascript/ecmascript-2015/string-flexibility/tagged-template-literals.md

```diff
@@ -44,43 +44,41 @@ aspects:
 ---
 ## Content
 
-ES6 introduces a new feature called template strings that make it easier to work with strings by adding string interpolation and multi-line strings:
+ES6 introduced a new feature called template strings that make it easier to work with strings by adding string interpolation and multi-line strings:
 
-```
+```js
 let company = "Enki";
```

Comment:
Add a comment here that explains that we're using string interpolation:

```js
// string interpolation
// is a fancy word for
// variable substitution

let company = "Enki";
console.log(`Hello ${company}!`);
//prints: "Hello Enki!" 
```

Reply:
done